language/Portuguese_Brazil/strings.po

```diff
@@ -20,871 +20,871 @@ msgstr ""
 
 #: src/emu/ui/auditmenu.cpp:111
 msgid "Audit in progress..."
-msgstr ""
+msgstr "Auditoria em progresso"
 
 #: src/emu/ui/barcode.cpp:72
 msgid "New Barcode:"
-msgstr ""
+msgstr "Novo c�digo de barras"
 
 #: src/emu/ui/barcode.cpp:76
 msgid "Enter Code"
-msgstr ""
+msgstr "Entrar com o c�digo"
 
 #: src/emu/ui/barcode.cpp:118
 msgid "Barcode length invalid!"
-msgstr ""
+msgstr "Tamanho do c�digo de barras inv�lido"
```

Comment:
again, here punctuation is not the same. You're forgetting the exclamation mark "!" in the end.